packages/web-react/src/components/Footer/README.md

```diff
@@ -0,0 +1,256 @@
+# Footer
+
+The Footer is a highly variable composition. It comes in several ready-to-use design variants, but can be customized to
+achieve your specific design goals.
+
+## Composition
+
+The Footer supports several ready-to-use building blocks:
+
+- [Navigation Links](#navigation-links)
+- [Product Logo](#product-logo)
+- [Social Media Links](#social-media-links)
+- [Language Switch](#language-switch)
+- [Secondary Links](#secondary-links)
+
+This is how all supported building blocks of the Footer build up the complete composition:
+
+```jsx
+<Footer UNSAFE_className="bg-cover pt-1100 pb-1000">
+  <Container>
+    {/* Grid with navigation links */}
+    {/* Grid with product logo, social media links and language switch */}
+    {/* Flex with secondary links */}
+  </Container>
+</Footer>
+```
+
+👉 Good to know:
+
+- To achieve the desired design, you can use the provided building blocks in any order you need.
+- All building blocks are optional, so you can use only the ones you need.
+- You are not limited just to the provided building blocks. You can add your own content or building blocks (such as a
+  newsletter subscription form) as needed.
+- Use [Grid][grid] and [Flex][flex] components to create the desired layout.
+- Use [Stack][stack] or [Divider][divider] components to organize individual sections.
+- Use [style props][readme-style-props] and utility classes like `mb-*` or `pt-*` to achieve desired spacings between components.
+
+## Navigation Links
+
+Navigation links are structured in sections with a headline and a [Stack][stack] of links.
+
+👉 Please note how the `<nav>` element is paired with an `<Text>` component (using the `aria-labelledby` attribute) to
+provide a semantic connection between the headline and the navigation component. Just make sure the `id` attribute of
+the `<Text>` component matches the value of the `aria-labelledby` attribute of the `<nav>` element and all `id`s are unique.
+
+```jsx
+<nav aria-labelledby="footer-navigation-section">
+  <Text id="footer-navigation-section" elementType="h3" size="large" emphasis="bold" marginBottom="space-600">
+    Section headline
+  </Text>
```

Comment:
This is a bit complicated… I can imagine a component like this:

```jsx
<FooterNavigationSection title="Section headline" titleId="footer-navigation-section">
  <Stack elementType="ul" spacing="space-500" hasSpacing>
    <li>
      <Link href="https://www.example.com">Link</Link>
    </li>
    <li>
      <Link href="https://www.example.com">Link</Link>
    </li>
    <li>
      <Link href="https://www.example.com">Link</Link>
    </li>
  </Stack>
</FooterNavigationSection>
```

Or with a subcomponent for the link use case:

```jsx
<FooterNavigationSection titleId="footer-navigation-section">
  <FooterNavigationSectionTitle id="footer-navigation-section">
    <Link href="#">Section headline</Link>
  </FooterNavigationSectionTitle>
  {/* … */}
</FooterNavigationSection>
```

What do you guys think? cc @crishpeen